# Linear Fit for Thruster Force to ESC pwm: y = mx + b
# All SLOPEs in [pwm/N], all YINTs in [pwm]
# Positive Thrust: Propeller moves fluid from the cone (bottom) towards the dome (top)
# CW: Defines thrusters in which CW rotation of propeller generates POSITIVE thrust
# CCW: Defines thrusters in which CCW rotation of propeller generates POSITIVE thrust
# NOTE: Three regions - Deadband, Startup (SU), and Primary
CCW:
  MIN_THRUST: 0.200000
  STARTUP_THRUST: 1.500000
  SU_POS_THRUST:
    SLOPE: -22.500000
    YINT: 1450
  SU_NEG_THRUST:
    SLOPE: -28.600000
    YINT: 1493
  POS_THRUST:
    SLOPE: -5.700000
    YINT: 1420
  NEG_THRUST:
    SLOPE: -7.500000
    YINT: 1527
CW:
  MIN_THRUST: 0.200000
  STARTUP_THRUST: 1.500000
  SU_POS_THRUST:
    SLOPE: 22.500000
    YINT: 1500
  SU_NEG_THRUST:
    SLOPE: 28.600000
    YINT: 1457
  POS_THRUST:
    SLOPE: 5.700000
    YINT: 1530
  NEG_THRUST:
    SLOPE: 7.500000
    YINT: 1423
ENABLE_SPL: true
ENABLE_SSL: true
ENABLE_SWF: true
ENABLE_SWA: true
ENABLE_HPF: true
ENABLE_HSF: true
ENABLE_HPA: true
ENABLE_HSA: true